### DISCUSSION TITLE
Too many notifications for received files

### DISCUSSION THREAD

**Discussion by Dikakus** [blueman-project/blueman#2045] [Ideas]
Hi. First of all, thank you for your work. This is a great tool to easily make bluetooth tasks.

I had to send a large number of picture files from my mobile to my rpi4 via bluetooth. The desktop windows was filled with too many file sending and receiving notifications, so I had to close all the windows one by one, wich was quite a tedious task. Maybe it could be a great idea to have a window with a list of receiving and received (maybe with a check icon) files. Even being able to open the file by double click.

File 1 ✓
File 2 ✓
File 3 ❌
File 4 ...

**Comment by infirit** [comment]
If you are seeing windows that means you are hitting our fallback. These normally are notifications. But I do agree (and have looked at this before) that reusing a window or notification would be beneficial in this scenario.

edit: The whole file transfer UX could use an overhaul
